C4SGWeb/src/app/project/list/project-list.component.html

```diff
@@ -1,16 +1,46 @@
-<!-- List Projects-->
+<!-- Search Projects-->
 <div class="container">
-<ul class="projects">
-    <li *ngFor="let project of projects" (click)="onSelect(project)" [class.selected]="project === selectedProject">
-        <tr>
-            <td><img src={{project.image}} height="200" width="200"></td>
-            <td>
-                <span><h5>{{project.name}}</h5></span><br>          
-                <span>{{project.organizationName}}</span><br>
-                <span>{{project.description}}</span><br>
-                <span>{{project.createTime}}</span><br>
-            </td>
-        </tr>
-    </li>
-</ul>
+  <div class="row center">
+    <div class="col s12 m8 offset-m2 l3 form-block">
+      <form>
+        <div class="row">
+          <div class="input-field col s12 m12 l12">
+            <input #projectName id="keyword" type="text">
+            <label for="keyword">Keyword</label>
+          </div>
+          <div class="col s12 m12 l12">
+            <button class="btn waves-effect waves-light" type="button"
+                    (click)="getProjectsByKeyword(projectName.value); projectName.value=''">Search
+            </button>
+          </div>
+        </div>
+      </form>
+    </div>
+    <!-- List Projects-->
+    <div class="col s12 l9 ">
+      <p *ngIf="projects">{{projects.length}} volunteer opportunities found</p>
+      <ul class="projects-block">
+        <li class="row" *ngFor="let project of projects" (click)="onSelect(project)"
+            [class.selected]="project === selectedProject">
+          <div class="col s4 m3 l3 card-image">
```

Comment:
I think 12 colums image is too huge for small screen. Here what it looks like on a screen of Nexus5
![untitled](https://cloud.githubusercontent.com/assets/16789689/23835684/c88d557e-077c-11e7-88c5-d087729eff80.png)